weaviate/collection/grpc.py

```diff
@@ -97,8 +97,32 @@ class LinkToMultiTarget(LinkTo):
 PROPERTIES = Union[List[Union[str, LinkTo]], str]
 
 # Can be found in the google.protobuf.internal.well_known_types.pyi stub file but is defined explicitly here for clarity.
-_StructValue: TypeAlias = Union[struct_pb2.Struct, struct_pb2.ListValue, str, float, bool, None]
-_PyValue: TypeAlias = Union[Dict[str, "_PyValue"], List["_PyValue"], str, float, bool, None]
+_StructValue: TypeAlias = Union[
+    struct_pb2.Struct,
+    struct_pb2.ListValue,
+    str,
+    float,
+    bool,
+    None,
+    List[float],
+    List[int],
+    List[str],
+    List[bool],
+    List[UUID],
+]
```

Comment:
Are the `List[...]` types here covered by the `struct_pb2.ListValue` type or are they different objects entirely? I added the `struct_pb2.ListValue` type to this assuming it covered `List` types, which are then converted to `List[...]` types in `_PyValue` as below.

Reply:
I get type warnings if I remove it, so Ia ssume it is not covered